.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -32,10 +32,16 @@ Fixes #
 
 **Does this PR introduce a user-facing change?**:
 <!--
-If no, just write "NONE" in the release-note block below.
-If yes, a release note is required:
-Enter your extended release note in the block below. If the PR requires additional action from users switching to the new release, include the string "action required".
+If yes, you should add a release note to changelog.md:
+
+Please write follow this format:
+<pr title> ([#<pr id>](<pr link>), [@<your github name>](<your github homepage link>))
+
+For example:
+Add tree to store copy of dependency's license ([#1847](https://github.com/kubeedge/kubeedge/pull/1847), [@kevin-wangzefeng](https://github.com/kevin-wangzefeng))
+
+The pr title should start with an uppercase letter.
 -->
```

Comment:
I am okay with this change, but all those comments are not shown the template,
see https://github.com/kubeedge/kubeedge/blob/c11266109adf97e050c208357ad64d6f16445d16/.github/PULL_REQUEST_TEMPLATE.md

Reply:
I have updated format.